executor/builder.go

```diff
@@ -1098,7 +1099,7 @@ func (b *executorBuilder) buildAnalyzeIndexPushdown(task plan.AnalyzeIndexTask)
 			Tp:             tipb.AnalyzeType_TypeIndex,
 			StartTs:        math.MaxUint64,
 			Flags:          statementContextToFlags(b.ctx.GetSessionVars().StmtCtx),
-			TimeZoneOffset: timeZoneOffset(b.ctx),
```

Comment:
@lamxTyler For analyse executor, do we need timezone name too?

Reply:
Do not need.

Reply:
@lamxTyler Is offset needed in analyzer?

Reply:
ping @lamxTyler

Reply:
Seems not needned.